examples/playground/src/configuration.js

```diff
@@ -57,5 +57,19 @@ export default {
   constants: {
     Tiles3DLoader,
     CesiumIonLoader
+  },
+
+  postProcessConvertedJson: json => {
+    // Filter out invalid props. Typically, props will be invalid while the user is typing.
```

Comment:
There are several ways to address this.
Perhaps we could make deck more resilient and not fail on one bad component...

Reply:
True. I figured it was for a bit of perf gains, and to keep things simpler. It could be argued that only something like this `json` module would need validation since here the end user is hooking into props directly whereas most apps have some abstraction in-between.

On the flip side, validation can be this simple.